client/client.go

```diff
@@ -398,13 +395,12 @@ func (c *Client) downloadTargets(role string) error {
 }
 
 func (c *Client) downloadSigned(role string, size int64, expectedSha256 []byte) ([]byte, *data.Signed, error) {
```

Comment:
`c.remote.GetMeta` is where the download actually happens so this very much does the download.